frontend/src/scenes/persons/Person.tsx

```diff
@@ -157,12 +163,30 @@ export function Person(): JSX.Element | null {
                     />
                 </TabPane>
                 <TabPane tab={<span data-attr="persons-events-tab">Events</span>} key={PersonsTabType.EVENTS}>
-                    <EventsTable
-                        pageKey={person.distinct_ids.join('__')} // force refresh if distinct_ids change
-                        fixedFilters={{ person_id: person.id }}
-                        showPersonColumn={false}
-                        sceneUrl={urls.person(urlId || person.distinct_ids[0] || String(person.id))}
-                    />
+                    {featureDataExploration ? (
+                        <Query
+                            query={{
+                                kind: NodeKind.DataTableNode,
```

Comment:
I think we would benefit greatly from knowing if a node is a presentation or data level node, just from the name.

However that assumption about being or not being a child isn't exactly correct. First, every node renders something. Pure JSON, but that's still something. So in a way, everything is a presentation-layer node. However the distinction is still relevant because only data-layer nodes fetch data (via `dataNodeLogic` no less). Presentation layer nodes are expected to use that data.

Regarding inheritance, the way I see it is:
- presentation nodes can have presentation nodes and data nodes as children
- data nodes can have data nodes as children

For a presentation node inside a presentation node scenario... well, dashboards.

I didn't want to impose any scheme yet, as work with porting insights over will surely multiply the options. However I'd suggest something like `` (nothing) and `Query` (data nodes). 

OK to punt for now?